src/pip/_internal/distributions/wheel.py

```diff
@@ -16,8 +17,15 @@ class WheelDistribution(AbstractDistribution):
 
     def get_pkg_resources_distribution(self):
         # type: () -> Distribution
-        return list(pkg_resources.find_distributions(
```

Comment:
It'd be nice to add a doc-string here. Possibly something like:

```
Load the metadata from the wheel file into memory and return a pkg_resources.Distribution that uses it.
```

Reply:
Updated [here](https://github.com/pypa/pip/pull/7539/files#diff-30d1f38ae8a3f2b0b04b1a948bb100acR20).